test/cppsuite/configs/operations_test_stress.txt

```diff
@@ -18,58 +18,58 @@ metrics_monitor=
 timestamp_manager=
 (
     enabled=true,
-    oldest_lag=5,
+    oldest_lag=30,
     op_rate=1s,
-    stable_lag=10
+    stable_lag=15
 ),
 workload_manager=
 (
     checkpoint_config=
     (
-        op_rate=30s,
+        op_rate=120s,
     ),
     populate_config=
     (
-        collection_count=500,
-        #5GB of data
-        key_count_per_collection=1000,
+        collection_count=200,
+        #200GB of data
+        key_count_per_collection=10000,
         key_size=100,
-        thread_count=20,
-        value_size=10000
+        thread_count=200,
+        value_size=100000
     ),
     insert_config=
     (
         key_size=100,
         op_rate=10ms,
-        ops_per_transaction=(max=20,min=10),
-        thread_count=25,
-        value_size=10000
+        ops_per_transaction=(max=20,min=0),
+        thread_count=40,
+        value_size=1000000
     ),
     read_config=
     (
-        op_rate=3ms,
-        ops_per_transaction=(max=100,min=50),
-        thread_count=20
+        op_rate=1ms,
+        ops_per_transaction=(max=1000,min=50),
+        thread_count=40
     ),
     remove_config=
     (
-        op_rate=100ms,
-        ops_per_transaction=(max=100,min=50),
-        thread_count=5
+        op_rate=1ms,
+        ops_per_transaction=(max=50,min=0),
+        thread_count=20
     ),
     update_config=
     (
         op_rate=10ms,
-        ops_per_transaction=(max=10,min=5),
-        thread_count=25,
-        value_size=10000
+        ops_per_transaction=(max=10,min=0),
+        thread_count=40,
+        value_size=1000000
     )
 ),
 operation_tracker=
 (
     # FIXME-WT-8640 Enable the tracker again once we have moved to a different host or modified the
```

Comment:
You should be able to remove the FIXME.
Did `s_mentions` tell you about this FIXME comment?

Reply:
Theres an issue `s_mentions` on my tree that makes it impossible to tell if it is throwing an error. I will create a jira to follow up.

Reply:
bde1106ec

Reply:
`s_mentions` cannot return an error, it simply prints stuff that you may want to look at.

Reply:
No I mean you can't tell if it genuinely found a problem, currently s_mentions dumps like 5 pages of information to my screen.

Reply:
Sounds really weird, please add your output to the ticket and I can have a look.